cpp/src/IceStorm/Replica.h

```diff
@@ -22,9 +22,9 @@ struct GroupNodeInfo
     // COMPILER FIX: Clang using libc++ requires to define operator=
     //
 #if defined(__clang__) && defined(_LIBCPP_VERSION)
-#ifdef ICE_CPP11_COMPILER
-    GroupNodeInfo(const GroupNodeInfo&) = default;
-#endif
+#   ifdef ICE_CPP11_COMPILER
+    GroupNodeInfo(const GroupNodeInfo&);
```

Comment:
Since you removed the `= default`, does it mean it should be `= delete`?

Reply:
I think `= default` is to force the generation of the copy constructor on C++17 mode or greater, we can probably keep this `=default` as this is internal, shouldn't be an issue for binary compatibility.